### ISSUE TITLE
USWDS - Disabled styles: Phase out disabled classes (Interim)

### ISSUE THREAD

**Issue by amyleadem** [uswds/uswds#5116]
Wherever possible, we should remove any classes that were created specifically for disabled styles. Instead, styles should be applied to both the `disabled` and `aria-disabled` attributes.
